eng/pipelines/templates/scripts/create-tags-and-git-release.yml

```diff
@@ -0,0 +1,17 @@
+parameters:
+  ArtifactLocation: ''
```

Comment:
Unless the value is valid to be empty I generally like to set the default value to be "not-specified" or something similar so that if someone does forget it they will get an error that indicates they didn't specify the value. It make diagnosing issues a little easier.